web/app.js

```diff
@@ -2085,7 +2085,7 @@ if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
     "https://mozilla.github.io",
   ];
   validateFileURL = function (file) {
-    if (file === undefined) {
```

Comment:
A slightly unrelated change, but the existing check seemed oddly specific given that we won't even try to open the document unless the `file`-parameter is actually defined:
https://github.com/mozilla/pdf.js/blob/2c135b02e0b80d961cc726c1d7e58c85f2fe54bc/web/app.js#L2252-L2258